### ISSUE TITLE
Remove Abort Button again and bring functionality to Cleaner Button

### ISSUE THREAD

**Issue by Tobias-B-Besemer** [bleachbit/wishlist#54]
I think it would be "smarter" if the Abort Button gets removed again and the Cleaner Button gets the Abort Button while a cleaning is running. (User needs IMHO just one button, because it's always just one functional, depending on what BB does.) I would mark this function change with a red X over the pic of the Cleaner Button...

**Comment by az0** [comment]
I agree about the two functions being mutually exclusive, but I think it could be confusing to have it change.

Also, the abort function is able to stop a preview, so this opens strange ideas such as clicking where the clean button once was to stop a preview. Also, should the preview button disappear while the preview is active?

**Comment by Tobias-B-Besemer** [comment]
Hadn't time to test latest builds in the last weeks... but I think if you will not bring the stop function to the start buttons, I wouldn't in your case spend ATM more time with the skinning! And no, I think the buttons should be marked as "not usable, ATM", but don't disappear!
